api/core/database.py

```diff
@@ -23,42 +23,51 @@ def save(self, email, raw_password):
         if self._id is None:
             db.session.add(self)
         else:
-            pass  # TBA change user email/password
+            pass
 
         db.session.commit()
         return self
 
-    @staticmethod
-    def _validate_mail(mail):
-        return bool(re.search(r"^[-\w.]+@([A-z0-9][-A-z0-9]+\.)+[A-z]{2,4}$", mail))
+    @classmethod
+    def verify_user(cls, email, password):
+        email = email.strip()
+        user = cls.query.filter_by(mail=email).first()
 
-    @staticmethod
-    def _validate_password(password):
-        return bool(8 <= len(password) <= 30)
+        if not User.query.filter_by(mail=email).first():
```

Comment:
Незачем делать запрос еще раз.
Давай упростим до:
```suggestion
        if not user:
```

Reply:
Resolved 8e3a7d2.